packages/docs/docs/docs/deploying.md

```diff
@@ -7,13 +7,11 @@ The following steps will get you started using ZeppelinOS.
 
 ## Install Node and NPM
 
-First, let's install [Node.js](http://nodejs.org/) and
-[npm](https://npmjs.com/). On their
-respective websites you will find specific instructions for your machine.
+First, install [Node.js](http://nodejs.org/) and [npm](https://npmjs.com/). On the respective websites you will find specific instructions for installation on your computer.
 
 ## Setting up your project
 
-We'll need to create a directory for our project and access it:
+Create a directory for your project and navigate into it:
```

Comment:
Why the change from first plural to second singular? I don't have a strong preference for either, but I'm curious about why the change.

Reply:
Sorry, I probably should have discussed this Slack. Using the first plural is fine. You see it more for general articles and blogs. I recommend going with second singular for a few reasons. Second singular is a style more preferred for company content, and it's more of a standard. I think addressing text to "you" is easier and more direct for giving instructions and guidance. But ultimately both are styles to choose from.